pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -0,0 +1,40 @@
+Summary: Catch-all configuration files for google-fluentd on GCE.
+Name: google-fluentd-catch-all-config
+Version: 0.1
+Release: 1%{?dist}
+License: Apache
+Group: System Environment/Daemons
+URL: https://cloud.google.com/logging/
+Requires: google-fluentd
+
+%description
+A set of configuration files for google-fluentd used to ingest logs
+from the system and third-party application packages.
+
+%files
+%config /etc/google-fluentd
+
+%clean
+# don't clean up the files here; we'll do it in the toplevel Makefile
```

Comment:
Yes, if I use the default %clean it does this:

+ cd /usr/local/google/home/salty/rpmbuild/BUILD
+ /bin/rm -rf /usr/local/google/home/salty/git/github/GoogleCloudPlatform/fluentd-catch-all-config/build/el/google-fluentd-catch-all-config-0.1/files

which is not what I want, since I then try to mv a file back which fails (and I also build a tarball out of that directory, although now that we're in the repo I may be able to drop tar support).  it's probably a consequence of the... "unorthodox" way I'm building the package - I populate that directory from the top level Makefile, not from the spec file, so I think that is why I don't want the "clean" step to happen here either.